ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -319,19 +333,20 @@ module.exports = Marionette.LayoutView.extend({
     return text
   },
   getFilters: function() {
-    var property = this.model.get('type')
-    var comparator = this.model.get('comparator')
-    var value = this.filterInput.currentView.model.getValue()[0]
+    const property = this.model.get('type')
+    const comparator = this.model.get('comparator')
+    const value = this.filterInput.currentView.model.getValue()[0]
+    const type = this.comparatorToCQL()[comparator]
 
     if (comparator === 'NEAR') {
       return CQLUtils.generateFilterForFilterFunction('proximity', [
         property,
         value.distance,
         value.value,
       ])
+    } else if (comparator === 'IS EMPTY') {
+      return CQLUtils.generateFilter(type, property, null)
```

Comment:
Can we add another function called `generateEmptyFilter` that we call instead of `generateFilter` so we can keep `generateAnyGeoFilter` from needing to know about this case?